### ISSUE TITLE
What happens when a first-time user tries to sign in on an offline device?

### ISSUE THREAD

**Issue by Murtaya** [MicrosoftDocs/windows-uwp#1157]
It's mentioned that:

> Sign in APIs will still be success when offline if the user has signed in once, and the last signed in account will be returned.

But can you post what the expectation is when a user who has never previously signed into an app/game tries to sign in when the device is offline?  Should the user not be allowed to get into the app/game?  I don't think there's anything the game or Xbox Live could really do in this case to continue, is there?

(I'm currently blocked on my game being certified because it does not allow first time users to sign in when the console is offline, and while this article hints that this might be ok or an expected outcome, it doesn't specifically state it)

Thanks!

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 767bdae4-2487-b156-e768-0e4615f443e2
* Version Independent ID: 80b63846-1156-49e5-508b-aab5df30806c
* Content: [Authentication for UWP projects - UWP app developer](https://docs.microsoft.com/en-us/windows/uwp/xbox-live/using-xbox-live/auth/authentication-for-uwp-projects)
* Content Source: [windows-apps-src/xbox-live/using-xbox-live/auth/authentication-for-UWP-projects.md](https://github.com/MicrosoftDocs/windows-uwp/blob/docs/windows-apps-src/xbox-live/using-xbox-live/auth/authentication-for-UWP-projects.md)
* Product: **uwp**
* GitHub Login: @aablackm
* Microsoft Alias: **aablackm**

**Comment by drewbatgit** [comment]
Thanks for taking the time to let us know about this and we sincerely apologize for the delayed response. Due to the sheer volume of feedback in our repo, we are currently prioritizing new Issues. If you believe your feedback is still actionable, let us know either directly in this thread or by filing a new Issue and we’ll work to incorporate the suggestions into the documentation. Though the Issue will be marked as Closed, I will monitor any replies that come afterwards so feel free to treat this like an open Issue. Many thanks!
